test/enigma_test.rb

```diff
@@ -3,10 +3,11 @@
 
 class EnigmaTest < MiniTest::Test
 
-  def test_it_exists
+  def test_it_exists_and_has_attributes
```

Comment:
Nice combination of tests here and remembering to add this after adding the instance variable